lab-beeksma/README.md

```diff
@@ -0,0 +1,12 @@
+# 401-js day 06 lab
+
+### About:
+  This project uses the net module of node.js to set up a TCP server and handle simple commands from connecting sockets.   When a socket connects to the server a client is created for that socket, with a unique id, and a default username.  Listeners are also set up for that socket listening for any data, error, or close. If data is passed the server will accept the commands '@all', '@dm', and '@nickname'.  
+  - '@all <message>' will send the message to every connected client.
+  - '@dm <nickname> <message>' will send the message to the specific client that has that nickname.
```

Comment:
Note the web rendering of this file parses your placeholders as HTML. You can either escape the \< with \\, or quote the whole string as \`code\`.